assembl/static/js/package-lock.json

```diff
@@ -6590,7 +6590,7 @@
       "dependencies": {
         "source-map": {
           "version": "0.4.4",
-          "resolved": "https://registry.npmjs.org/source-map/-/source-map-0.4.4.tgz",
+          "resolved": "http://registry.npmjs.org/source-map/-/source-map-0.4.4.tgz",
```

Comment:
This is strange, I commited the version with https some days ago because I had the change when running fabric.

Reply:
on our side it keeps on rolling back to http: when we run `fabric`... something that we don't really encounter using yarn.